editor/src/components/inspector/constraints-section.spec.browser2.tsx

```diff
@@ -207,4 +210,30 @@ describe('Constraints Section', () => {
       }),
     )
   })
+
+  it('is hidden when there are no contents', async () => {
```

Comment:
I know this is going to seem a bit pedantic, but I think this (and the PR title and description) are a bit misleading here. The reason that the contents was empty was because of the types of the selected elements, so now we're just not showing the entire section based on that (rather than explicitly checking if the section is empty and hiding it if so). What I mean by that is we could later end up in a situation where the constraints section is empty but still shown. So really this test should be `'is hidden when _ elements are selected'`.

Reply:
It's true, the PR is hiding the constraints section if there are no displayable portions of the section. The reason is partly because of how the e.g. pins are detected (there's a bunch of "ifs" that need to be put in place to make sure the pins mean "empty" - if that's even a case). I probably should rename the related code to something like what you suggested ^